src/scenes/Wallet/components/ActionButtonHeader/index.js

```diff
@@ -1,15 +1,13 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
 import { injectIntl, defineMessages } from 'react-intl';
 import { withStyles, Button, Tooltip } from '@material-ui/core';
-
+import { inject, observer } from 'mobx-react';
```

Comment:
put this in the top imports. the order goes: external imports -> project imports

Reply:
@dwalintukan oh, i c. seems like the linter rule for this gets disabled?

Reply:
@henryzbw technically its fine. just a nitpick that the external imports was grouped with the project imports.